core/ustring.cpp

```diff
@@ -3349,11 +3349,17 @@ bool String::is_valid_identifier() const {
 
 	const wchar_t *str = &operator[](0);
 
+	if (len == 1 && str[0] == '_') {
```

Comment:
Should it check for greater than zero or am I missing something?

Reply:
No, since `"_xx"` is fine, just `"_"` causes some problems with GDScript.